pkgs/top-level/all-packages.nix

```
@@ -22746,6 +22746,8 @@ in
 
   meh = callPackage ../applications/graphics/meh {};
 
+  mediaelch = libsForQt514.callPackage ../applications/misc/mediaelch { };
```

```suggestion
  mediaelch = libsForQt5.callPackage ../applications/misc/mediaelch { };
```
(Unless it doesn't build with 5.15.)